app/models/user.rb

```diff
@@ -79,7 +86,22 @@ def password_reset_expired?
   end
 
   def feed
-    microposts
+    Micropost.where("user_id IN (?) OR user_id = ?", following_ids, id)
```

Comment:
viết scope cho where này nhé

ngoài ra, ko cần or trong này, chỉ cần gộp giá trị của `following_ids, id` là dc
`following_ids << id`

scope là ->(user_ids){where "user_id IN ?", users_ids}`